apps/datahub/src/app/home/search/search-summary/search-summary.component.html

```diff
@@ -1,20 +1,51 @@
-<div>
-  <gn-ui-results-hits></gn-ui-results-hits>
-</div>
-<div class="mt-4" *ngIf="publisher$ | async as publisher">
-  <div translate="">search.filters.active</div>
-  <div
-    class="inline-block bg-primary py-2 px-3 rounded font-medium text-gray-50 text-sm leading-none mt-2"
-  >
-    <div class="flex">
-      <div
-        class="close bg-white bg-opacity-50 text-primary w-4 cursor-pointer h-4 text-xl leading-3 hover:bg-opacity-100 rounded-full text-center transition-all"
-        style="line-height: 14px"
-        (click)="removeOrg()"
+<div
+  class="px-3 pt-6 sm:p-7 sm:my-8 sm:shadow-primary-light sm:rounded-xl grid grid-cols-3 gap-4 transition-[height] duration-100"
+  [ngClass]="
+    isOpen ? 'bg-primary text-white h-[204px]' : 'bg-white text-main h-[98px]'
+  "
+>
+  <div class="col-span-2 grid grid-cols-2 gap-4 grid-rows-auto">
+    <div class="col-span-2 flex flex-row items-baseline" *ngIf="isOpen">
+      <h2 class="text-2xl mr-4" translate>search.filters.title</h2>
+      <gn-ui-results-hits class="text-sm opacity-[60%]"></gn-ui-results-hits>
+    </div>
+    <gn-ui-filter-dropdown
+      class="w-full"
+      [fieldName]="'OrgForResource'"
+      [title]="'search.filters.byOrganisation' | translate"
+    ></gn-ui-filter-dropdown>
+    <gn-ui-filter-dropdown
+      class="w-full"
+      [fieldName]="'linkProtocol'"
+      [title]="'search.filters.byFormat' | translate"
+    ></gn-ui-filter-dropdown>
+    <gn-ui-filter-dropdown
```

Comment:
There seems to be a bug with the dropdown that is initially hidden (not sure if related to this fact). The first value checked in the dropdow is not added to the selected list and thus can not be unchecked afterwards.

Reply:
Hmm, could not reproduce... can we try to see if that happens on DEV? might be related to something in your local environment

Reply:
Hum, weird, it only appears for the geographic scale on my side. Let's test on dev, then :-)

Reply:
Ok, I've reproduced and fixed the issue, it was because the possible values were numerical!